core/server/master/src/test/java/alluxio/master/file/meta/MountTableTest.java

```diff
@@ -390,6 +391,35 @@ public void getMountInfo() throws Exception {
     Assert.assertEquals(null, mMountTable.getMountInfo(4L));
   }
 
+  /**
+   * Tests {@link MountTable#translate(String)}.
+   */
+  @Test
+  public void translate() throws Exception {
```

Comment:
nit: split UT into smaller cases , one UT case for a single test case (e.g., successful translate, unknown translation). perhaps also the case without scheme (treated as alluxio path as documented in FSM interface)

Reply:
Actually, `MountTable#translate` requires foreign URI. So can't test scheme-less URIs here.
Split the UT and changed method name to reverseLookup which makes more sense for mount table.